src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/FindOneAndDeleteCommandResolver.java

```diff
@@ -69,7 +69,8 @@ private FindOperation getFindOperation(
           DocumentProjector.includeAllProjector(),
           ReadType.DOCUMENT,
           objectMapper,
-          vector);
+          vector,
+          false);
```

Comment:
So `findOneAndXxx()` can never return/include sort vector, even if one was used?

Reply:
Checked the spec, only `find()` and `findOne()` support it.